internal/convert/rs_config.go

```diff
@@ -77,10 +78,20 @@ func (c *RSConfig) patchConfigServer(out *opsmngr.AutomationConfig, name string)
 // protocolVer determines the appropriate protocol based on FCV
 // returns "0" for versions <4.0 or "1" otherwise
 func (c *RSConfig) protocolVer() (string, error) {
-	if c.FCVersion == "" {
-		return "", nil
+	fcVersion := c.FCVersion
+	if fcVersion == "" {
+		// search per process, this may be the case whe users do a get first and update it
```

Comment:
```suggestion
		// search per process, this may be the case when users do a get first and update it
```

[q] Could you explain a bit more what you mean here with "get first and update it"? Just to make clear what's the scenario here

Reply:
updated